pkg/3scale/amp/component/system.go

```diff
@@ -47,6 +47,16 @@ const (
 	SystemSecretSystemRedisURLFieldName  = "URL"
 	SystemSecretSystemRedisSentinelHosts = "SENTINEL_HOSTS"
 	SystemSecretSystemRedisSentinelRole  = "SENTINEL_ROLE"
+
+	// TLS env vars
+	SystemSecretSystemRedisCAFile            = "REDIS_CA_FILE"
```

Comment:
I feel like I've seen this above already - if re-used, would it make more sense to have it in a single file as a single source of truth?